README.md

```diff
@@ -121,10 +121,10 @@ Join a channel:
 /join [channel]
 ```
 
-Start a direct chat with someone:
+Start a direct chat with someone or multiple users:
 ```
-/query [username]
-/slack talk [username]
+/query [username] ([username2] [username3]...)
```

Comment:
The help in weechat for `/query` says:
`[irc]  /query  [-noswitch] [-server <server>] <nick>[,<nick>...] [<text>]`

Usually, I think we should stick to the same syntax for our commands as irc uses, so separate the nicks with comma instead of space. The only thing that's making me unsure is that tab completion doesn't seem to be working when using comma.

Reply:
Yeah I was trying to decide between spaces and commas also, and went with spaces because of tab completion haha. I didn't know that weechat's help specified commas though, so I think it makes sense to stick with their syntax instead.